ios/Integreat.xcodeproj/xcshareddata/xcschemes/integreat-test-cms.xcscheme

```diff
@@ -99,7 +99,7 @@
       buildConfiguration = "Debug">
    </AnalyzeAction>
    <ArchiveAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
integreat-test-cms has a Releas buildConfiguration? Is that correct?

Reply:
I think so @klinzo right?

Reply:
Yep